### DISCUSSION TITLE
Feature Request: Disable Boost at individual account level

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#20021] [General]
It would help keep the Home timeline less cluttered if we could select the accounts for which we don't want to see boosts.

I don't want to turn boosts off across the board because some people are boosting interesting content, but others are basically spamming with the rate they boost.

ETA: feature add to the Mastodon app (android) and mobile browser.

**Comment by Sirablopp** [comment]
Afaik that option is already in there, at least in the web version. Go to the profile of that user, click the three dots next to the "follow"-button and select the respective option. (I am not sure what it is named in English, in German it's "Geteilte Beiträge von (user) entfernen".) At least I think that's how it's supposed to work, I never used it.

If it is not available in the app, I agree this feature should be added there. :)

**Reply by ghost** [reply]
Ah, okay you're right, it is available on the desktop web version. The feature currently is not available on either the main Mastodon app (android) or the mobile browser version.

I'll use the desktop version to update some of the more boost-happy folks I'm following. Thank you!
